src/quo/components/list/item.cljs

```diff
@@ -4,40 +4,50 @@
             [quo.haptic :as haptic]
             [quo.gesture-handler :as gh]
             [quo.design-system.spacing :as spacing]
-            [quo.design-system.colors :as colors]
-            [quo.components.text :as text]
+            [quo.design-system.colors :as quo.colors]
+            [clojure.string :as string]
             [quo.components.controls.view :as controls]
             [quo.components.tooltip :as tooltip]
+            [quo2.components.markdown.text :as text]
             [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
             [quo.components.animated.pressable :as animated]))
 
+(defn- icons-lib [icon-name props]
```

Comment:
hmm, I believe we agreed to copy code like this into quo2 and then avoid this sort of new ui / old ui branching.

Reply:
I mean moving the item component. I guess we need to decide on a structure for where we migrate pieces over to.

ay the moment we have:
```
src/quo2
 |-components
 |-foundations	
 |- gesture.cljs
 |- reanimated.cljs	
 |- screens

```
Maybe we can rename `src/quo2/components` to `src/quo2/ui-components` (for components that are Figma only) so that we can directly migrate over components such as list item etc ?? and also have a place for these components that are more foundational and not in the Figma files
@OmarBasem , @jo-mut @ilmotta , @Parveshdhull , @erikseppanen. any thoughts?

Reply:
@J-Son89 What do you mean by foundational components? I think all the components we create come from Figma. Other basic components which are the RN imported components are currently kinda scattered in the codebase. I was talking earlier with Andrey that we need to have them in one place (probably outside our component library that we constantly edit).

Reply:
Thanks Omar for the response :) Yeah, by foundational components I am also referring to basic components which are the RN imported components, and also components such as the "item component" here which is not part of our quo2 component library (i.e there's no visual demo) but is used to create the quo2 components.

Reply:
I agree with you guys. To me it seems pretty convenient to just move (copy actually) namespaces like `quo/react-native.cljs` to `quo2/`, that's what we already did by putting there namespaces such as `quo2.gesture` and `quo2.reanimated`, which are not Figma components, they're more like UI utility namespaces.

Reply:
My point here also is that to make this migration of pieces from `quo` to `quo2` is that we can what currently is `quo2/components` to something like `quo2/figma-components` so that we can just do a direct migration of `quo` pieces  as `quo/components` exists and contains these utility type components

Reply:
@J-Son89 I believe that we are supposed to implement list items under `quo2/components/list_items`. That's what Andrey told me.

Reply:
The hierarchy is overall kinda messy. We have components dir under `quo`/`quo2` and also under `ui`/`ui2`. List items are in different places, as well as RN comps.

Reply:
yep, I agree. It would be nice to put some more thought into it and clean this up while we are in the migration process